### ISSUE TITLE
Missing information dot on page next to themes

### ISSUE THREAD

**Issue by RowinRuizendaal** [RowinRuizendaal/Bloom#92]

**Comment by ralfz123** [comment]
Content has to be in Vuex store? Like the themes content.

**Comment by ralfz123** [comment]
## layout done
<img width="380" alt="Screenshot 2021-07-13 at 17 34 08" src="https://user-images.githubusercontent.com/55750121/125481184-fb36bce8-e81d-45ef-933b-ac8bc797a84f.png">

**Comment by RowinRuizendaal** [comment]
Yes,  please set content in store, that way we have all the content at one place
